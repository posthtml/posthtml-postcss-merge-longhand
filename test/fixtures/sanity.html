<div style="margin-top: 1px; margin-right: 2px; margin-bottom: 3px; margin-left: 4px;">margin</div>
<div style="padding-top: 1px; padding-right: 2px; padding-bottom: 3px; padding-left: 4px;">padding</div>
<div style="border-color: red; border-width: 1px; border-style: dashed;">border</div>
<div style="background-image: url('{{ $foo }}');">expressions</div>
