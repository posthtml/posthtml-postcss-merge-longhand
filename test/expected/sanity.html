<div style="margin: 1px 2px 3px 4px;">margin</div>
<div style="padding: 1px 2px 3px 4px;">padding</div>
<div style="border: 1px dashed red;">border</div>
<div style="background-image: url('{{ $foo }}');">expressions</div>
